ui/src/jobs/details/config/components/bq_options_table/BigQueryOptionsTable.js

```diff
@@ -11,8 +11,7 @@ export const BigQueryOptionsTable = ({ options }) =>
         title,
         description,
       }))}
-      titleProps={{ style: { width: "30%" } }}
-      descriptionProps={{ style: { width: "70%" } }}
+      columnWidths={[1, 7/3]}
```

Comment:
Replacing the soon to be deprecated props `titleProps` and `descriptionProps` with the new `columnWidths`. The functionality is similar and I'm simply using the same ratio as before.